include/uxr/agent/middleware/ced/CedMiddleware.hpp

```diff
@@ -28,123 +28,339 @@ class CedMiddleware : public Middleware
     CedMiddleware() = default;
     ~CedMiddleware() override = default;
 
-/**********************************************************************************************************************
- * Create functions.
- **********************************************************************************************************************/
+    /**
+     * @startuml
+     * !include agent/middleware/ced/create_participant.puml!0
+     * @enduml
+     *
+     * @brief Creates a CedParticipant from a reference.
+     *        Currently, only the domain_id is taken into account for the creation.
+     *        In a near future, the Middleware interface should change the ref parameter by an "attributes" one.
+     *        For this purpose an XML parser and a reference data base are needed in the XRCE Core.
+     * @param participant_id    The CedParticipant identifier.
+     * @param domain_id         The Domain identifier to which the CedParticipant belongs.
+     * @param ref               The CedParticipant reference (not used).
```

Comment:
I would do (not used) more outstanding, `NOT USED`?

Reply:
Fixed!